cmd/certsuite/upload/results_spreadsheet/results_spreadsheet.go

```diff
@@ -0,0 +1,367 @@
+	if err != nil {
+		return nil, err
+	}
+
+	if err := addFilterByFailedAndMandatoryToSheet(sheetService, workloadResultsSpreadsheet, "results"); err != nil {
+		return nil, err
+	}
+
+	if err := MoveSpreadSheetToFolder(driveService, folder, workloadResultsSpreadsheet); err != nil {
+		return nil, err
+	}
+
+	return workloadResultsSpreadsheet, nil
+}
+
+//nolint:funlen
+func createConclusionsSheet(sheetsService *sheets.Service, driveService *drive.Service, rawResultsSheet *sheets.Sheet) (*sheets.Sheet, error) {
+	headers := getHeadersFromSheet(rawResultsSheet)
+	colsIndices, err := getHeaderIndicesByColumnNames(headers, []string{"CNFName", "CNFType", "OperatorVersion"})
+	if err != nil {
+		return nil, err
+	}
+
+	workloadNameColIndex := colsIndices[0]
+	workloadTypeColIndex := colsIndices[1]
+	operatorVersionColIndex := colsIndices[2]
+
+	// Initialize sheet with headers
+	conclusionsSheetRowsValues := []*sheets.CellData{}
+	for _, colHeader := range conclusionSheetHeaders {
+		headerCellData := &sheets.CellData{UserEnteredValue: &sheets.ExtendedValue{StringValue: stringToPointer(colHeader)}}
+		conclusionsSheetRowsValues = append(conclusionsSheetRowsValues, headerCellData)
+	}
+	conclusionsSheetRows := []*sheets.RowData{{Values: conclusionsSheetRowsValues}}
+
+	// Extract unique values from the CNFName column and fill sheet
+	uniqueMap := make(map[string]bool)
+	for _, rawResultsSheetrow := range rawResultsSheet.Data[0].RowData[1:] {
+		workloadName := *rawResultsSheetrow.Values[workloadNameColIndex].UserEnteredValue.StringValue
+		// if workload has already been added to sheet, skip it
+		if uniqueMap[workloadName] {
+			continue
+		}
+		uniqueMap[workloadName] = true
+
+		curConsclusionRowValues := []*sheets.CellData{}
+		for _, colHeader := range conclusionSheetHeaders {
+			curCellData := &sheets.CellData{UserEnteredValue: &sheets.ExtendedValue{}}
+			var value string
+
+			switch colHeader {
+			case categoryConclusionsCol:
+				value = *rawResultsSheetrow.Values[workloadTypeColIndex].UserEnteredValue.StringValue
+
+			case workloadVersionConclusionsCol:
+				value = *rawResultsSheetrow.Values[operatorVersionColIndex].UserEnteredValue.StringValue
+
+			case ocpVersionConclusionsCol:
+				value = ocpVersion + " "
+
+			case workloadNameConclusionsCol:
+				value = workloadName
+
+			case resultsConclusionsCol:
+				workloadResultsSpreadsheet, err := createSingleWorkloadRawResultsSpreadSheet(sheetsService, driveService, workloadsResultsFolder, rawResultsSheet, workloadName)
+				if err != nil {
+					return nil, fmt.Errorf("error has occurred while creating %s results file: %v", workloadName, err)
+				}
+
+				hyperlinkFormula := fmt.Sprintf("=HYPERLINK(%q, %q)", workloadResultsSpreadsheet.SpreadsheetUrl, "Results")
+				curCellData.UserEnteredValue.FormulaValue = &hyperlinkFormula
+
+			default:
+				// use space for empty values to avoid cells overlapping
+				value = " "
+			}
+
+			if colHeader != resultsConclusionsCol {
```

Comment:
It is worth having a comment here on why this column/cell is filtered now and when it's going to be set.

Reply:
Now I see I was wrong here, since the value was already set as a formula value. Sorry for the noise :(

Reply:
though I think my change made it clearer, take a look: https://github.com/redhat-best-practices-for-k8s/certsuite/blob/9ad972f30706b8c6a4dadafc457166dd1eeaa0d4/cmd/certsuite/upload/results_spreadsheet/results_spreadsheet.go#L260